app/models/claim.rb

```diff
@@ -74,6 +74,10 @@ def additional_defendants
     defendants.where(main: false)
   end
 
+  def gdpr_documents_deleted?
+    false
+  end
```

Comment:
should there be a comment here to explain that this can default as false on the model because it would only ever not be false once the claim is in the app store (therefore we wouldn't be using this logic, but using the logic in the app store claim presenter)

Reply:
comment added 👍🏼